public/json/hyper_key.json

```
@@ -0,0 +1,31 @@
+{
+	"title": "Hyper Key",
```

```suggestion
	"title": "Caps Lock → Hyper Key (⌃⌥⇧⌘) (Caps Lock if alone)",
```

"Hyper Key" is too vague for a title, when considering, complex modifications here are meant to be imported by others. :-)